scout/server/blueprints/panels/templates/panels/panel.html

```diff
@@ -232,6 +238,33 @@
   </div>
 {% endmacro %}
 
+{% macro case_hits_export_modal() %}
```

Comment:
Good to have from the panel page, but when navigating from the case page this could be bypassed or at least defaulted to the current case and institute?